Makefile

```diff
@@ -3,33 +3,18 @@
 # SPDX-License-Identifier: MPL-2.0
 ts := $(shell date +"%Y%m%d%H%M")
 
-binary:
-		nix-build -A tezos-client-babylonnet -o tezos-client
-		cp tezos-client/tezos-client-babylonnet-* tezos-client-babylonnet
-		rm tezos-client
+binaries:
+		nix-build -A babylonnet-binaries -o binaries
+		cp binaries/* ./
+		rm binaries
 
-binary-mainnet:
-		nix-build -A tezos-client-mainnet -o tezos-client
-		cp tezos-client/tezos-client-mainnet-* ./tezos-client-mainnet
-		rm tezos-client
+binaries-mainnet:
+		nix-build -A mainnet-binaries -o binaries
+		cp binaries/* ./
+		rm binaries
 
-rpm:
-		nix-build -A babylonnet-rpm-package -o tezos-client-package --arg timestamp $(ts)
-		cp -a tezos-client-package/*.rpm .
-		rm tezos-client-package
-
-rpm-mainnet:
-		nix-build -A mainnet-rpm-package -o tezos-client-package --arg timestamp $(ts)
-		cp -a tezos-client-package/*.rpm .
-		rm tezos-client-package
-
-deb:
-		nix-build -A babylonnet-deb-package -o tezos-client-package --arg timestamp $(ts)
-		cp -a tezos-client-package/*.deb .
-		rm tezos-client-package
-
-deb-mainnet:
-		nix-build -A mainnet-deb-package -o tezos-client-package --arg timestamp $(ts)
-		cp -a tezos-client-package/*.deb .
-		rm tezos-client-package
+rpm-packages:
```

Comment:
you produce the output `make` expects but don't specify any build inputs. Should be phony.

Reply:
I agree that `binaries` doesn't produce a file named `binaries`, but this file produces symlink named `rpm-packages` as well as `deb-packages`